wagtail/contrib/redirects/templates/wagtailredirects/choose_import_file.html

```diff
@@ -0,0 +1,43 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% block titletag %}{% trans "Redirects" %}{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+    <script>
+        window.headerSearch = {
+            url: "{% url 'wagtailredirects:index' %}",
+            termInput: "#id_q",
+            targetOutput: "#redirects-results"
+        }
+    </script>
+{% endblock %}
+
+{% block content %}
+    {% trans "Import redirects" as import_str %}
+    {% include "wagtailadmin/shared/header.html" with title=import_str icon="redirect" %}
+
+    {% if form.non_field_errors %}
```

Comment:
I think this form, on initial load, needs a little more info for the user.

It is unclear that you will be taken to a next step that will let you validate/approve/ see errors, it is also unclear that the format used can be any header name (allocatable later).

I know that there are docs underway, however there could be a little more help given to the user.

One idea would be to add a help block here - for example:
```
    <div class="nice-padding">
        <div class="help-block help-info">
            {% blocktrans %}
                <p>Select a file that contains a column with header <pre>from</pre> and <pre>to</pre>, with each row being one redirect.</p>
                <p>... file formats are available.</p>
            {% endblocktrans %}
        </div>
    </div>
```

Alternatively, adding some help text to the import file field might be enough, something along the lines of 'once submitted, you will be given the change to select the from/to mapping and confirm before adding'.

Reply:
What I did here was to add a info block on top and requirements in the file uploader help text.